### PR TITLE
Added project by findingone

### PR DESCRIPTION
**Pull request by Findingone**

### CODE REVIEW

File: projects/Findingone/a2oj-Implementation.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - Frontend Developers
+  - DevOps
+  - Designers
+  - Maintainers
+  - Code Reviewers
+  - Technical Writers
+  - Mentors
+  - Testers
+  - UX
+  - (you may add custom entries, but please also add them to the canonical list in project/_template.mdx)
```
Comment (deammer) [inline projects/Findingone/a2oj-Implementation.mdx:28]: **issue (blocking):** this is the default content list in the template. Please fill out this information with content relevant to your project!

File: projects/Findingone/a2oj-Implementation.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+contributionOverview: # (optional) Provide additional context to help contributors match well with your project
+  mainLocation: Europe
+  idealEffort: 1 PR a month
+  isMentorshipAvailable: true
+  automatedDevEnvironment: https://www.gitpod.io/
+learnLinks: # (optional) A list of links to support new contributors in learning about your project and tech stack and culture / community
+  - title: Learn Gatsby
+    url: https://www.gatsbyjs.com/
+  - title: Get started with Markdown
+    url: https://www.markdownguide.org/getting-started/
```
Comment (deammer) [inline projects/Findingone/a2oj-Implementation.mdx:41]: **issue (blocking):** this is also the content from the template. Please update it accordingly.

File: projects/Findingone/a2oj-Implementation.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+<!-- Divide your content into sections using our handy shortcodes. Each section is optional -->
+
+<Overview>
+
+Curated cp problems divided into ladders that increase in difficulty as you progress.
+
+**Hint:** you can write GitHub-flavored markdown just like you're used to. We even support Slack-style emojis :v:
```
Comment (deammer) [inline projects/Findingone/a2oj-Implementation.mdx:50]: **issue (blocking):** this is the default content from the template as well. Please update it accordingly!

We recommend explaining the purpose of the project, what goals are you trying to achieve, how to get started as a contributor, etc.

**Review by deammer — CHANGES_REQUESTED** [review]

**Comment by ramonh** [comment]
Closing due to inactivity.

**Pull request closed without merging**